GitUI/UserControls/FileStatusList.cs

```diff
@@ -804,7 +803,9 @@ public async Task OpenSubmoduleAsync()
 
             var submoduleName = SelectedItem.Item.Name;
 
-            var status = await SelectedItem.Item.GetSubmoduleStatusAsync().ConfigureAwait(false);
+            Task<GitSubmoduleStatus?>? task = SelectedItem.Item.GetSubmoduleStatusAsync();
+            Validates.NotNull(task);
```

Comment:
This could probably be rewritten to not validate, but it got too complicated to bother...

Reply:
Removed the Validates (if task or status is null there are no commits to select in the subrepo)